# M743T KI modeling

<!-- Below was Taken from V1.0 - Protocols: CRISPR/Cas9 (with HD) zebrafish zygote microinjection protocol -->

## 13Nov2019 CRISPR/Cas9 (with HD) zebrafish zygote microinjection protocol:

<!-- \hypertarget{protocol1}{protocol number 1} -->

**ssODN dilution:**

Stock ssODN (M743T HD) is concentrated to 5932 ng/µg, dilution is required for injection mix preperation.

Mix material by Table 1's instructions on ice. Afterwards carry out vortex and spindown.

**Table 1 - ssODN dilution:**

```{r, echo=FALSE}

library(knitr)
library(kableExtra)
Material<-"dr_GNE_HDR"
Initial.Conc<-"5932 ng/µL"
DDW.Volume<- "11 µL"
Volume.To.Pick<-"1 µL"
Final.Concentration<-"12 µL"

DF<-data.frame(Material, Initial.Conc, DDW.Volume, Volume.To.Pick, Final.Concentration)
kable(DF, "latex", booktabs = TRUE)
```


**Cas9 protein dilution:**

Stock Cas9 (protein) is concentrated to 5000 ng/µg, dilution is required for injection mix preperation.

Mix material by Table 2's instructions on ice. Afterwards carry out vortex and spindown.

**Table 2:**

| Material | Initial Concentration | Volume to pick | DDW  | final concontration | Final Volume |
|----------+-----------------------+----------------+------+---------------------+--------------|
| Cas9     | 5000 ng/µL            | 1 µL           | 4 µL | 1000 ng/µg          | 12 µL        |

**Cas9/gRNA complex formation**

Mix:

* DDW from Table 3 microinjection mix.
* Cas9 from Table 3 microinjection mix.
* gRNA from Table 3 microinjection mix.

Place in incubation at 37°C for ~10 miutes.

**Injection mix:**

On ice, mix the following materials. Perform vortex and spindown.

**Table 3:**
```{r, table 3, echo = FALSE}
Material <- c("Cas9", "gRNA",
             "ssODN", "KCl",
             "DDW", "final volume")
Vol.To.Pick<- c(1,1,1,0.5,1.5,5)
Initial.Conc<- c(1000, 1020.75, 494.999, 3000, "-", "-")
Final.Concentration<-c(200,204.15,98.866,300,"-","-")

df<-data.frame(Material,Vol.To.Pick,Initial.Conc,Final.Concentration)

library(knitr)
library(kableExtra)
kable(df, "latex", booktabs = TRUE) %>%
  kable_styling(latex_options = "striped")
```

**Scr7 Dilution:**

In a 50 ml falcon tube:

**Table 4:**

```{r, echo=FALSE}

library(knitr)
library(kableExtra)
Material<-"Scr7"
Initial.Conc<-"4 mM"
DDW.Volume<- "300 µL"
Volume.To.Pick<-"30 mL"
Final.Concentration<-"20 µM"

DF<-data.frame(Material, Initial.Conc, DDW.Volume, Volume.To.Pick, Final.Concentration)
kable(DF, "latex", booktabs = TRUE)
```

**DMSO Dilution:**

In a 50 ml falcon tube:

**Table 5:**

```{r, echo=FALSE}

library(knitr)
library(kableExtra)
Material<-"DMSO"
Initial.Conc<-"X mM"
DDW.Volume<- "300 µL"
Volume.To.Pick<-"30 mL"
Final.Concentration<-"20 µM"

DF<-data.frame(Material, Initial.Conc, DDW.Volume, Volume.To.Pick, Final.Concentration)
kable(DF, "latex", booktabs = TRUE)
```

**Injection:**

* After zf mate - TL type, injection volume ~2 nL of injection mix (table 3), inject embryos at 1-cell stage on agarose gel track plate (pre heated to 28.5°C).
* Keep a few embryos uninjected for negative control.
* After injection move to Scr7 medium.


## 13Nov2020 Microinjections:

*Date: November 13^th^, 2019*
Following protocol <!-- \hyperlink{protocol1}{{\color{purple}{\underline{number 1}}}}. -->

We got our first batch which I now call Batch number 1.

### Batch table:

```{r, echo = FALSE}
library(tibble)
library(kableExtra)
batch.file<-tribble(
  ~Batch, ~Parents, ~`Date`, ~`Number of offsprings`, ~`Number of living offsprings`,
  "Batch 1", "TL find out which cell" ,"Nov 13 2019", 40 , 18
)
kable(batch.file, "latex", booktabs = TRUE)
```

Batch 1 compiles of 40 zygotes, 27 of which were microinjected by Hagai, 13 of which were injected by me.
See [my plate](images/batch1TL_M743T_MI_Tom_nov_13_2019.jpg), or [Hagai's plate](images/batch1TL_M743T_MI_hagai_nov_13_2019.jpg)

Due to distance between our lab and lab 3 where the microinjections took place, along with unawareness of lack of devices such as 10 µL pipettes, we performed the injections on 8-16 cell stage zygotes (Hagai and I).

In my case some of the microinjections did not enter the cell, most were inside the egg close to the cell.






### Microinjection expertise examination Tom V Hagai

*Date: December 10^th^, 2019*
After conducting Microinjections on November 13^th^, I have data about larvae's survivability after injection. Part of the experiment's objectives was to examine my skills vs Hagai's skills. 

[Raw data - my table](tables/Tom_VS_Hagai_R1_Tom.csv)

[Raw data - Hagai's table](tables/Tom_VS_Hagai_R1_Hagai.csv)

### Table 7:

```{r, echo=FALSE}
library(tibble)
library(kableExtra)
DF1<- read.csv("tables/Tom_VS_Hagai_R1_Tom.csv") # My table
DF2<-read.csv("tables/Tom_VS_Hagai_R1_Hagai.csv") # Hagai's table
DF1$Dead.so.far<-with(DF1, 13-Live.Larvae) # made a new column calculate how many larvae died at that time
DF1[1,4]<- NA # Making first cell of dead so far empty
DF2$Dead.so.far<-with(DF2, 27-Live.Larvae)  # made a new column calculate how many larvae died at that time
DF2[1,4]<- NA # Making first cell of dead so far empty
# end of dataframe import
DF1[1,4]<- as.numeric(DF1[1,4])
DF2[1,4]<- as.numeric(DF2[1,4])
y<-c()
for (i in 1:length(DF1[,4])) {
  y<- append(y, (sum(as.numeric(DF1[2:i,4]))))
}
# y 
DF1$Accumulative.Dead <- y
s<- c()
for (i in 1:length(DF1[,4])) {
  s<- append(s, (as.numeric(DF1[i,5])/as.numeric(DF1[8,5]))*100)
}
# s
DF1$accu.dead.perc<-s

# Hagai
y<-c()
for (i in 1:length(DF2[,4])) {
  y<- append(y, (sum(as.numeric(DF2[2:i,4]))))
}
# y 
DF2$Accumulative.Dead <- y

s<- c()
for (i in 1:length(DF2[,4])) {
  s<- append(s, (as.numeric(DF2[i,5])/as.numeric(DF2[8,5]))*100)
}
# s
DF2$accu.dead.perc<-s

# End of making an accumulative column of dead %

# Naming headers and adding timeline for x axis ----
colnames(DF1)<- c("Date checked", 
                  "Time of the day", 
                  "Live count", 
                  "Dead Count", 
                  "Dead accumulated", 
                  "Dead accumulated percentage")
colnames(DF2)<- c("Date checked", 
                  "Time of the day", 
                  "Live count", 
                  "Dead Count", 
                  "Dead accumulated", 
                  "Dead accumulated percentage")
DF1$Time<- c(0,7,17,24,41,48,55,72)
DF2$Time<- c(0,7,17,24,41,48,55,72)

# Plotting ---- 

# plot(DF1[,7],DF1[,6], type = "b", main = "Tom's microinjection \naccumulative death count by percentage",
#      xlab = "time passed since injection",
#      ylab = "Percent of accumulated dead zf larvae" )

# plot(DF2[,7],DF2[,6], type = "b", main = "Hagai's microinjection \naccumulative death count by percentage",
#      xlab = "time passed since injection",
#      ylab = "Percent of accumulated dead zf larvae" )

# statistical

# summary(DF1)
# summary(DF2)
kable(DF1, booktabs = TRUE)
```

### Table 8:
```{r, echo=FALSE}
library(tibble)
library(kableExtra)
DF1<- read.csv("tables/Tom_VS_Hagai_R1_Tom.csv") # My table
DF2<-read.csv("tables/Tom_VS_Hagai_R1_Hagai.csv") # Hagai's table
DF1$Dead.so.far<-with(DF1, 13-Live.Larvae) # made a new column calculate how many larvae died at that time
DF1[1,4]<- NA # Making first cell of dead so far empty
DF2$Dead.so.far<-with(DF2, 27-Live.Larvae)  # made a new column calculate how many larvae died at that time
DF2[1,4]<- NA # Making first cell of dead so far empty
# end of dataframe import
DF1[1,4]<- as.numeric(DF1[1,4])
DF2[1,4]<- as.numeric(DF2[1,4])
y<-c()
for (i in 1:length(DF1[,4])) {
  y<- append(y, (sum(as.numeric(DF1[2:i,4]))))
}
# y 
DF1$Accumulative.Dead <- y
s<- c()
for (i in 1:length(DF1[,4])) {
  s<- append(s, (as.numeric(DF1[i,5])/as.numeric(DF1[8,5]))*100)
}
# s
DF1$accu.dead.perc<-s

# Hagai
y<-c()
for (i in 1:length(DF2[,4])) {
  y<- append(y, (sum(as.numeric(DF2[2:i,4]))))
}
# y 
DF2$Accumulative.Dead <- y

s<- c()
for (i in 1:length(DF2[,4])) {
  s<- append(s, (as.numeric(DF2[i,5])/as.numeric(DF2[8,5]))*100)
}
# s
DF2$accu.dead.perc<-s

# End of making an accumulative column of dead %

# Naming headers and adding timeline for x axis ----
colnames(DF1)<- c("Date checked", 
                  "Time of the day", 
                  "Live count", 
                  "Dead Count", 
                  "Dead accumulated", 
                  "Dead accumulated percentage")
colnames(DF2)<- c("Date checked", 
                  "Time of the day", 
                  "Live count", 
                  "Dead Count", 
                  "Dead accumulated", 
                  "Dead accumulated percentage")
DF1$Time<- c(0,7,17,24,41,48,55,72)
DF2$Time<- c(0,7,17,24,41,48,55,72)

# Plotting ---- 

# plot(DF1[,7],DF1[,6], type = "b", main = "Tom's microinjection \naccumulative death count by percentage",
#      xlab = "time passed since injection",
#      ylab = "Percent of accumulated dead zf larvae" )

# plot(DF2[,7],DF2[,6], type = "b", main = "Hagai's microinjection \naccumulative death count by percentage",
#      xlab = "time passed since injection",
#      ylab = "Percent of accumulated dead zf larvae" )

# statistical

# summary(DF1)
# summary(DF2)
kable(DF2, "latex", booktabs = TRUE)
```
### Graph 1:
```{r, echo=FALSE}
library(tibble)
library(kableExtra)
DF1<- read.csv("tables/Tom_VS_Hagai_R1_Tom.csv") # My table
DF2<-read.csv("tables/Tom_VS_Hagai_R1_Hagai.csv") # Hagai's table
DF1$Dead.so.far<-with(DF1, 13-Live.Larvae) # made a new column calculate how many larvae died at that time
DF1[1,4]<- NA # Making first cell of dead so far empty
DF2$Dead.so.far<-with(DF2, 27-Live.Larvae)  # made a new column calculate how many larvae died at that time
DF2[1,4]<- NA # Making first cell of dead so far empty
# end of dataframe import
DF1[1,4]<- as.numeric(DF1[1,4])
DF2[1,4]<- as.numeric(DF2[1,4])
y<-c()
for (i in 1:length(DF1[,4])) {
  y<- append(y, (sum(as.numeric(DF1[2:i,4]))))
}
# y 
DF1$Accumulative.Dead <- y
s<- c()
for (i in 1:length(DF1[,4])) {
  s<- append(s, (as.numeric(DF1[i,5])/as.numeric(DF1[8,5]))*100)
}
# s
DF1$accu.dead.perc<-s

# Hagai
y<-c()
for (i in 1:length(DF2[,4])) {
  y<- append(y, (sum(as.numeric(DF2[2:i,4]))))
}
# y 
DF2$Accumulative.Dead <- y

s<- c()
for (i in 1:length(DF2[,4])) {
  s<- append(s, (as.numeric(DF2[i,5])/as.numeric(DF2[8,5]))*100)
}
# s
DF2$accu.dead.perc<-s

# End of making an accumulative column of dead %

# Naming headers and adding timeline for x axis ----
colnames(DF1)<- c("Date checked", 
                  "Time of the day", 
                  "Live count", 
                  "Dead Count", 
                  "Dead accumulated", 
                  "Dead accumulated percentage")
colnames(DF2)<- c("Date checked", 
                  "Time of the day", 
                  "Live count", 
                  "Dead Count", 
                  "Dead accumulated", 
                  "Dead accumulated percentage")
DF1$Time<- c(0,7,17,24,41,48,55,72)
DF2$Time<- c(0,7,17,24,41,48,55,72)

# Plotting ---- 

plot(DF1[,7],DF1[,6], type = "b", main = "Tom's microinjection \naccumulative death count by percentage",
     xlab = "time passed since injection",
     ylab = "Percent of accumulated dead zf larvae" )

# plot(DF2[,7],DF2[,6], type = "b", main = "Hagai's microinjection \naccumulative death count by percentage",
#      xlab = "time passed since injection",
#      ylab = "Percent of accumulated dead zf larvae" )

# statistical

# summary(DF1)
# summary(DF2)

```

### Graph 2:

```{r, echo=FALSE}
library(tibble)
library(kableExtra)
DF1<- read.csv("tables/Tom_VS_Hagai_R1_Tom.csv") # My table
DF2<-read.csv("tables/Tom_VS_Hagai_R1_Hagai.csv") # Hagai's table
DF1$Dead.so.far<-with(DF1, 13-Live.Larvae) # made a new column calculate how many larvae died at that time
DF1[1,4]<- NA # Making first cell of dead so far empty
DF2$Dead.so.far<-with(DF2, 27-Live.Larvae)  # made a new column calculate how many larvae died at that time
DF2[1,4]<- NA # Making first cell of dead so far empty
# end of dataframe import
DF1[1,4]<- as.numeric(DF1[1,4])
DF2[1,4]<- as.numeric(DF2[1,4])
y<-c()
for (i in 1:length(DF1[,4])) {
  y<- append(y, (sum(as.numeric(DF1[2:i,4]))))
}
# y 
DF1$Accumulative.Dead <- y
s<- c()
for (i in 1:length(DF1[,4])) {
  s<- append(s, (as.numeric(DF1[i,5])/as.numeric(DF1[8,5]))*100)
}
# s
DF1$accu.dead.perc<-s

# Hagai
y<-c()
for (i in 1:length(DF2[,4])) {
  y<- append(y, (sum(as.numeric(DF2[2:i,4]))))
}
# y 
DF2$Accumulative.Dead <- y

s<- c()
for (i in 1:length(DF2[,4])) {
  s<- append(s, (as.numeric(DF2[i,5])/as.numeric(DF2[8,5]))*100)
}
# s
DF2$accu.dead.perc<-s

# End of making an accumulative column of dead %

# Naming headers and adding timeline for x axis ----
colnames(DF1)<- c("Date checked", 
                  "Time of the day", 
                  "Live count", 
                  "Dead Count", 
                  "Dead accumulated", 
                  "Dead accumulated percentage")
colnames(DF2)<- c("Date checked", 
                  "Time of the day", 
                  "Live count", 
                  "Dead Count", 
                  "Dead accumulated", 
                  "Dead accumulated percentage")
DF1$Time<- c(0,7,17,24,41,48,55,72)
DF2$Time<- c(0,7,17,24,41,48,55,72)

# Plotting ---- 

# plot(DF1[,7],DF1[,6], type = "b", main = "Tom's microinjection \naccumulative death count by percentage",
#      xlab = "time passed since injection",
#      ylab = "Percent of accumulated dead zf larvae" )

plot(DF2[,7],DF2[,6], type = "b", main = "Hagai's microinjection \naccumulative death count by percentage",
     xlab = "time passed since injection",
     ylab = "Percent of accumulated dead zf larvae" )

# statistical

# summary(DF1)
# summary(DF2)

```


### DF1: 

```{r, echo=FALSE}
library(tibble)
library(kableExtra)
DF1<- read.csv("tables/Tom_VS_Hagai_R1_Tom.csv") # My table
DF2<-read.csv("tables/Tom_VS_Hagai_R1_Hagai.csv") # Hagai's table
DF1$Dead.so.far<-with(DF1, 13-Live.Larvae) # made a new column calculate how many larvae died at that time
DF1[1,4]<- NA # Making first cell of dead so far empty
DF2$Dead.so.far<-with(DF2, 27-Live.Larvae)  # made a new column calculate how many larvae died at that time
DF2[1,4]<- NA # Making first cell of dead so far empty
# end of dataframe import
DF1[1,4]<- as.numeric(DF1[1,4])
DF2[1,4]<- as.numeric(DF2[1,4])
y<-c()
for (i in 1:length(DF1[,4])) {
  y<- append(y, (sum(as.numeric(DF1[2:i,4]))))
}
# y 
DF1$Accumulative.Dead <- y
s<- c()
for (i in 1:length(DF1[,4])) {
  s<- append(s, (as.numeric(DF1[i,5])/as.numeric(DF1[8,5]))*100)
}
# s
DF1$accu.dead.perc<-s

# Hagai
y<-c()
for (i in 1:length(DF2[,4])) {
  y<- append(y, (sum(as.numeric(DF2[2:i,4]))))
}
# y 
DF2$Accumulative.Dead <- y

s<- c()
for (i in 1:length(DF2[,4])) {
  s<- append(s, (as.numeric(DF2[i,5])/as.numeric(DF2[8,5]))*100)
}
# s
DF2$accu.dead.perc<-s

# End of making an accumulative column of dead %

# Naming headers and adding timeline for x axis ----
colnames(DF1)<- c("Date checked", 
                  "Time of the day", 
                  "Live count", 
                  "Dead Count", 
                  "Dead accumulated", 
                  "Dead accumulated percentage")
colnames(DF2)<- c("Date checked", 
                  "Time of the day", 
                  "Live count", 
                  "Dead Count", 
                  "Dead accumulated", 
                  "Dead accumulated percentage")
DF1$Time<- c(0,7,17,24,41,48,55,72)
DF2$Time<- c(0,7,17,24,41,48,55,72)

# Plotting ---- 

# plot(DF1[,7],DF1[,6], type = "b", main = "Tom's microinjection \naccumulative death count by percentage",
#      xlab = "time passed since injection",
#      ylab = "Percent of accumulated dead zf larvae" )

# plot(DF2[,7],DF2[,6], type = "b", main = "Hagai's microinjection \naccumulative death count by percentage",
#      xlab = "time passed since injection",
#      ylab = "Percent of accumulated dead zf larvae" )

# statistical

summary(DF1)
# summary(DF2)

```


### DF2:

```{r, echo=FALSE}
library(tibble)
library(kableExtra)
DF1<- read.csv("tables/Tom_VS_Hagai_R1_Tom.csv") # My table
DF2<-read.csv("tables/Tom_VS_Hagai_R1_Hagai.csv") # Hagai's table
DF1$Dead.so.far<-with(DF1, 13-Live.Larvae) # made a new column calculate how many larvae died at that time
DF1[1,4]<- NA # Making first cell of dead so far empty
DF2$Dead.so.far<-with(DF2, 27-Live.Larvae)  # made a new column calculate how many larvae died at that time
DF2[1,4]<- NA # Making first cell of dead so far empty
# end of dataframe import
DF1[1,4]<- as.numeric(DF1[1,4])
DF2[1,4]<- as.numeric(DF2[1,4])
y<-c()
for (i in 1:length(DF1[,4])) {
  y<- append(y, (sum(as.numeric(DF1[2:i,4]))))
}
# y 
DF1$Accumulative.Dead <- y
s<- c()
for (i in 1:length(DF1[,4])) {
  s<- append(s, (as.numeric(DF1[i,5])/as.numeric(DF1[8,5]))*100)
}
# s
DF1$accu.dead.perc<-s

# Hagai
y<-c()
for (i in 1:length(DF2[,4])) {
  y<- append(y, (sum(as.numeric(DF2[2:i,4]))))
}
# y 
DF2$Accumulative.Dead <- y

s<- c()
for (i in 1:length(DF2[,4])) {
  s<- append(s, (as.numeric(DF2[i,5])/as.numeric(DF2[8,5]))*100)
}
# s
DF2$accu.dead.perc<-s

# End of making an accumulative column of dead %

# Naming headers and adding timeline for x axis ----
colnames(DF1)<- c("Date checked", 
                  "Time of the day", 
                  "Live count", 
                  "Dead Count", 
                  "Dead accumulated", 
                  "Dead accumulated percentage")
colnames(DF2)<- c("Date checked", 
                  "Time of the day", 
                  "Live count", 
                  "Dead Count", 
                  "Dead accumulated", 
                  "Dead accumulated percentage")
DF1$Time<- c(0,7,17,24,41,48,55,72)
DF2$Time<- c(0,7,17,24,41,48,55,72)

# Plotting ---- 

# plot(DF1[,7],DF1[,6], type = "b", main = "Tom's microinjection \naccumulative death count by percentage",
#      xlab = "time passed since injection",
#      ylab = "Percent of accumulated dead zf larvae" )

# plot(DF2[,7],DF2[,6], type = "b", main = "Hagai's microinjection \naccumulative death count by percentage",
#      xlab = "time passed since injection",
#      ylab = "Percent of accumulated dead zf larvae" )

# statistical

# summary(DF1)
summary(DF2)

```
<!--Taken from V1.0 under High prioritya-->

## 03Feb2019 A1 DNA extraction & PCR for sequencing, TL AB exon 3 seq

*February 3^rd^, 2020*


[INSERT THIS](documents/Scan_Feb_3_2020.pdf)

### Exon 3 sequencing

Project students require exon 3 sequence of AB and TL ZF to modify their reations based on our strains' SNPs. 

* Single AB^wt^ individual from Jan 30^th^, 2020 breeding; AB genotype - \textcolor{red}{check whith hagai which parents}. 
* Single TL^wt^, jan 26^th^, 2020 breeding; D8 parents. 

Their positions on the strip [FCD#001 3.2] are:


| 1 | 2  | 3  | 4 | 5 | 6 | 7 | 8 |
|--:+----+----+---+---+---+---+---|
| x | AB | TL | x | x | x | x | x |

### Larvae DNA extraction protocol:

For larvae samples.

Place larvae in tube and let it fall asleep cooling on ice.

Drain water with pipettor.

Add 50 [µl] lysis buffer.

Incubate for 10 [min] at 95 [°C]. 

Add 2.5 [µl] Protein K.

Incubate for 2-16 [h] at 55 [°C].

Cool on ice.

- Placed the tubes in our lab's PCR device. \textcolor{red}{need to collect tomorrow}. 

### A1 FC PCR - PCRFC#001

Again, for project students, these are the samples they tested nested PCR on (inner R primer specific for my KI design).
I do believe these samples are from details established at 2017 and not my details.

* Took a 1:10 dilution from stock DNA extract made by Hagai on FC from A1 details (Are now in genotype, \textcolor{red}{write where they are now in genotype aquarium})
* The following PCR established:

PCR mix:

|Content| 1X volume [µl]| 8X volume [µl]| Concentration|
|:------|:-------------:|:--------:|:--:|
|DNA | 1| 8.2||
| F1 primer (M743T outer)| 1| 8.2||
| R1 primer (M743T outer)| 1| 8.2||
| GoTaq | 12.5| 102.5||
| DDW| 9.5| 77.9||
| Total| 25| 205||



insert concentrations

PCR conditions: Insert PCR conditions

In tubes:

|1|2 |3 |4|5|6|7|8|
|:-:|:-:|:-: |:-:|:-:|:-:|:-:|:-:|
|A1-1|A1-2|A1-3|A1-4|A1-5|A1-6|A1-7|NC|

### Preparations for sequencing:

Preparing primers for sequencing:

|Name|DDW [µl]|100 nM (?) Primer Volume[µl]| Total| Label for sequencers|
|:---|:---:|:---:|:-|:-:|
|ex12 345 R|47.4|2.5|50| R1|
|ex12 1F|47.4|2.5|50|F1|

## 07Jan2020 gRNA IVT (/w GNE 2020 project team):


**INSERT PROTOCOL**

**MISTAKES**

* No DNase was added before RNA production(word?) was made. Gel expresses the presence of template DNA. 

**DATA to collect:**

* Materials, cas and other catalog numbers.
* Number of gRNA tubes.
* Gel run-time:
  * 70 min, 110V
  * 2 8 well gels
  * 0.495g TopVision
  * 0.459g hy-labs
  * 33 ml TBEX0.5 
  * Gelred
  * 5 µl loaded to gel


**Files:**
* [gRNA Gel 1 Image 0.5µl IVT in 2.5µl DDW and 3µl kit buffer](images/Master 2020-01-07_17h42m09s gRNA high conc.jpg)
* [gRNA Gel 1 Image 0.5µl IVT in 5.5µl DDW, then 3µl of that with 3µl kit buffer](images/Master 2020-01-07_17h42m09s gRNA low conc.jpg)
* [Nanodrop results](tables/jan07_2020_IVTgRNA_Nanodrop.xml)


## 10June2020 Fin clips, 19-20 May 2020 - Lab procedure and results

### PCR preparation:

* Made a working stock for `ex12_345R` reverse primer for exon 12 amplification.
* 10 $\mu M$ from $10 \mu l$ of $100 \mu M$ stock and $90 \mu l$ DDW.
* I will PCR only strips: A-D + Z.
* A-D PCR by WTS only (need to perform F1_2 for heterozygotes screening).
* Strips A-D + Z were boiled at $95 $ for 10 minutes.
* Mixes for ex12 and WTS amplifications were prepared according to 9June2020 preparation tables.
* DNA extract were dilute 1/40 from stock.
* DNA for Z was taken from a 1/10 dilution. 

## PCR strip 

A: A1-A8
B: B1-B5, 6=NTC, B7-B8
C: C1-C8
D: D1-D7
Z: Z1-Z3, Z6-Z8, 7=NTC, 8=-

### PCR programs:

`Users>HL>28WTSF`
`System>Tom>Ex12345`

### Results:

**Strips A,B,C**
<!-- fix img pos
```{r, Gel1, out.width = "300 pt"}
knitr::include_graphics("../../365/10June2020/Master 2020-06-10_15h42m20s.png")
```
-->
**Strips D,Z**
<!-- fix img pos
```{r, Gel2, out.width = "300 pt"}
knitr::include_graphics("../../365/10June2020/Master 2020-06-10_15h45m46s.png")
```
-->
Both gels: 1.5%, 120V @ 28min. 

### Stored:

* PCR amplicons inside Molecular lab (big) at $-20° fridge.
* DNA extracts in our lab's fridge.

### Follow up:

* Analyze results.
* Repair PCR for strip Z, send to sequence. 
* PCR strip Y.
* PCR A-D+Y, F1_2.
* Move Fish accordingly.
* Repair latex code for tables 9June2020....
(Image for DNA extract box at daily folder)





## 11June2020 Incubator Documentation 24well plates


### Dish A from 2June2020


9June2020 is 7 dpf, n=43
From which 7 died and sampled, 8 were taken to "poor" plate, rest left. 
10June2020 eve, n=39


Today I replaced water and counted $n=38$.
1 detail was moved to small dish for poor A larvae (new) (Movie_453.avi) plate markings: 


-----
A 11June2020 Poor details
n=1
-----




### Dish B From 2June2020:


Offspring tracking.
9June2020 dfp = 7, n = 28
10June2020 dpf = 8, n = 27
10 "poor" details were separated to plate B.








Easy peasy no one showed any lack of money.
1 dead on lid, taken to new strip S1 at position 1
| 1                          | 2 | 3 | 4 | 5 | 6 | 7 | 8 |
|----------------------------+---+---+---+---+---+---+---|
| 11.6 From big dish B (2.6) |   |   |   |   |   |   |   |


### Big dish C From 3June2020


Cross +/- X +/- from #6
9june2020 6dpf n=54




11june2020 - n=48 (reafirm)
fed and water replaced




### 24plate B:


#### no.55 from 8june2020
9june n=55
10june n=39
11june:
n=39
no gfp


#### no.48 from 8june2020
n=60
9june =55
10june n=55
11june:
n=55


#### inj3 09june2020


09june n= 38
09june n=31
10june n=25
11june:
n=23


2 details look really poor were separated


#### inj3 control (no injection) no.79 9june2020
n0=79
9june n=7
10jun n=6
11june all still in chorion, n=6


#### inj2 8june2020
n0=38
11june2020 n=
21


#### inj2 control 8june2020
n0 = 10
11june n=8

\pagebreak{}{}

## 15Jun2020 Nu7447

Nu777 arrived look at arts an prepare stock.
Store at 4 deg C.

## 16Jun2020 CRISPR/Cas9 MI KI

Inj by protocol no.

Parents TL wt from system assorted tank no. 22 gave first batch of eggs. 33 of which injected, 6 put aside for control.

Parents TLwt from SAT no. 24 gave a bad batch, 2nd batch 42 inj (24 live), 3rd batch missed (4 cell stg) of which 49 put aside for control.


Took me 36 min to prep mix

FAILED - NO BREADING


### Inj ovservation and treatment:


From:  inj 15june2020 1dpf

• no. 22 CR n=13
• no.22 control n=3
• no. 24 CR n=3
• no.24 Control n=9

Time of treatment: 11:00

### Inj ovservation and treatment:

• no. 22 CR n=13
• no.22 control n=3
• no. 24 CR n=2
1 sampled to **strip** (16.6 sCR AAA1) pos. 1 @ 20 µl buffer lysys. 
• no.24 Control n=9

Time of treatment: 18:08



## 19June2020 Fin clips, 19-20 May 2020 - Planning

Fin clip for mut zfGNE and model ZF from May 2020 
**Introduction:** During 19-20 may 2020 we collected fin flips from M743T KI ZF, mut zfGNE and hGNE zf KO models. 

**Strip Z contained KI ZF as follows**

| Strip position | Expected genotype | Fish physical position |
| :-------------:|-------------------|:----------------------:|
| 1              |  M743T            | A1                     |
| 2              |  M743T            | A2                     |
| 3              |  M743T            | A3                     |
| 4              |  Alon alpha actin |                        |
| 5              |  Alon alpha actin |                        |
| 6              |  M743T            | A6                     |
| 7              |  M743T            | A7                     |
| 8              |  M743T            | A8                     |

Even though we took FC of 4 and 5 (strip positions) we won't sequence them since there's no need to, we do need to breed them though. 

**strip y goes as follows:**

| strip position  | expected genotype          | fish physical position   |
| :-------------: | -------------------------- | :----------------------: |
| 1               | AB tg mut Cross TL zfGNE   | A9                       |
| 2               | AB tg mut Cross TL zfGNE   | A10                      |
| 3               | AB tg mut Cross TL zfGNE   | A11                      |
| 4               |                            |                          |
| 5               |                            |                          |
| 6               |                            |                          |
| 7               |                            |                          |
| 8               |                            |                          |

Beyond that we also got Strips A-D which contain FC from 15 dpf larvae that urgently need to be placed into a container.

\newpage
\blandscape

```{r, echo=FALSE}
x<-read.csv('tables/FCmaypoorlarvae.csv')
library(knitr)
library(kableExtra)
kable(x, "latex", booktabs = TRUE) %>%
  kable_styling(latex_options = "striped")
```


\elandscape

\pagebreak{}{}

### PCR mixes:

**For Strip Z:**

```{r, echo=FALSE}
x<-read.csv('tables/9junePCRmix.csv')
library(knitr)
library(kableExtra)
kable(x, "latex", booktabs = TRUE) %>%
  kable_styling(latex_options = "striped")
```

**For the rest:**

**WTS:**
```{r, echo=FALSE}
x<-read.csv('tables/9june2020PCRtableWTS.csv')
library(knitr)
library(kableExtra)
kable(x, "latex", booktabs = TRUE) %>%
  kable_styling(latex_options = "striped")
```
**F1_2**
```{r, echo=FALSE}
x<-read.csv('tables/9june2020PCRF1_2.csv')
library(knitr)
library(kableExtra)
kable(x, "latex", booktabs = TRUE) %>%
  kable_styling(latex_options = "striped")
```

### Dna Extract

• DNA extract from yesterday were put in - 20 deg.
