<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>A Minimal Book Example</title>
  <meta name="description" content="This is a minimal example of using the bookdown package to write a book. The output format for this example is bookdown::gitbook." />
  <meta name="generator" content="bookdown 0.21 and GitBook 2.6.7" />

  <meta property="og:title" content="A Minimal Book Example" />
  <meta property="og:type" content="book" />
  
  
  <meta property="og:description" content="This is a minimal example of using the bookdown package to write a book. The output format for this example is bookdown::gitbook." />
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="A Minimal Book Example" />
  
  <meta name="twitter:description" content="This is a minimal example of using the bookdown package to write a book. The output format for this example is bookdown::gitbook." />
  

<meta name="author" content="Yihui Xie" />


<meta name="date" content="2020-10-31" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  

<link rel="next" href="june2020-fin-clips-19-20-may-2020-lab-procedure-and-results.html"/>
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />









<link href="libs/anchor-sections-1.0/anchor-sections.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.0/anchor-sections.js"></script>


<style type="text/css">
a.sourceLine { display: inline-block; line-height: 1.25; }
a.sourceLine { pointer-events: none; color: inherit; text-decoration: inherit; }
a.sourceLine:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode { white-space: pre; position: relative; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
code.sourceCode { white-space: pre-wrap; }
a.sourceLine { text-indent: -1em; padding-left: 1em; }
}
pre.numberSource a.sourceLine
  { position: relative; left: -4em; }
pre.numberSource a.sourceLine::before
  { content: attr(title);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; pointer-events: all; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {  }
@media screen {
a.sourceLine::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
</style>

<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">A Minimal Book Example</a></li>

<li class="divider"></li>
<li class="chapter" data-level="1" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i><b>1</b> Prerequisites</a><ul>
<li class="chapter" data-level="1.1" data-path="index.html"><a href="index.html#nov2019-crisprcas9-with-hd-zebrafish-zygote-microinjection-protocol"><i class="fa fa-check"></i><b>1.1</b> 13Nov2019 CRISPR/Cas9 (with HD) zebrafish zygote microinjection protocol:</a></li>
<li class="chapter" data-level="1.2" data-path="index.html"><a href="index.html#microinjections-on-november-13th"><i class="fa fa-check"></i><b>1.2</b> Microinjections on November 13<sup>th</sup>:</a><ul>
<li class="chapter" data-level="1.2.1" data-path="index.html"><a href="index.html#batch-table"><i class="fa fa-check"></i><b>1.2.1</b> Batch table:</a></li>
</ul></li>
<li class="chapter" data-level="1.3" data-path="index.html"><a href="index.html#microinjection-expertise-examination-tom-v-hagai"><i class="fa fa-check"></i><b>1.3</b> Microinjection expertise examination Tom V Hagai</a><ul>
<li class="chapter" data-level="1.3.1" data-path="index.html"><a href="index.html#table-7"><i class="fa fa-check"></i><b>1.3.1</b> Table 7:</a></li>
<li class="chapter" data-level="1.3.2" data-path="index.html"><a href="index.html#table-8"><i class="fa fa-check"></i><b>1.3.2</b> Table 8:</a></li>
<li class="chapter" data-level="1.3.3" data-path="index.html"><a href="index.html#graph-1"><i class="fa fa-check"></i><b>1.3.3</b> Graph 1:</a></li>
<li class="chapter" data-level="1.3.4" data-path="index.html"><a href="index.html#graph-2"><i class="fa fa-check"></i><b>1.3.4</b> Graph 2:</a></li>
<li class="chapter" data-level="1.3.5" data-path="index.html"><a href="index.html#statistical-analysis"><i class="fa fa-check"></i><b>1.3.5</b> Statistical analysis</a></li>
</ul></li>
<li class="chapter" data-level="1.4" data-path="index.html"><a href="index.html#a1-dna-extraction-pcr-for-sequencing-tl-ab-exon-3-seq"><i class="fa fa-check"></i><b>1.4</b> A1 DNA extraction &amp; PCR for sequencing, TL AB exon 3 seq</a></li>
<li class="chapter" data-level="1.5" data-path="index.html"><a href="index.html#exon-3-sequencing"><i class="fa fa-check"></i><b>1.5</b> Exon 3 sequencing</a><ul>
<li class="chapter" data-level="1.5.1" data-path="index.html"><a href="index.html#larvae-dna-extraction-protocol"><i class="fa fa-check"></i><b>1.5.1</b> Larvae DNA extraction protocol:</a></li>
<li class="chapter" data-level="1.5.2" data-path="index.html"><a href="index.html#a1-fc-pcr---pcrfc001"><i class="fa fa-check"></i><b>1.5.2</b> A1 FC PCR - PCRFC#001</a></li>
<li class="chapter" data-level="1.5.3" data-path="index.html"><a href="index.html#preparations-for-sequencing"><i class="fa fa-check"></i><b>1.5.3</b> Preparations for sequencing:</a></li>
</ul></li>
<li class="chapter" data-level="1.6" data-path="index.html"><a href="index.html#grna-ivt-w-gne-2020-project-team"><i class="fa fa-check"></i><b>1.6</b> gRNA IVT (/w GNE 2020 project team):</a><ul>
<li class="chapter" data-level="1.6.1" data-path="index.html"><a href="index.html#gel-electroporesis-results-for-a5-details-pcr-for-exon-13"><i class="fa fa-check"></i><b>1.6.1</b> Gel Electroporesis Results For A5 details PCR for exon 13</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="2" data-path="june2020-fin-clips-19-20-may-2020-lab-procedure-and-results.html"><a href="june2020-fin-clips-19-20-may-2020-lab-procedure-and-results.html"><i class="fa fa-check"></i><b>2</b> 10June2020 Fin clips, 19-20 May 2020 - Lab procedure and results</a><ul>
<li class="chapter" data-level="2.1" data-path="june2020-fin-clips-19-20-may-2020-lab-procedure-and-results.html"><a href="june2020-fin-clips-19-20-may-2020-lab-procedure-and-results.html#pcr-preparation"><i class="fa fa-check"></i><b>2.1</b> PCR preparation:</a></li>
<li class="chapter" data-level="2.2" data-path="june2020-fin-clips-19-20-may-2020-lab-procedure-and-results.html"><a href="june2020-fin-clips-19-20-may-2020-lab-procedure-and-results.html#pcr-strip"><i class="fa fa-check"></i><b>2.2</b> PCR strip</a></li>
<li class="chapter" data-level="2.3" data-path="june2020-fin-clips-19-20-may-2020-lab-procedure-and-results.html"><a href="june2020-fin-clips-19-20-may-2020-lab-procedure-and-results.html#pcr-programs"><i class="fa fa-check"></i><b>2.3</b> PCR programs:</a></li>
<li class="chapter" data-level="2.4" data-path="june2020-fin-clips-19-20-may-2020-lab-procedure-and-results.html"><a href="june2020-fin-clips-19-20-may-2020-lab-procedure-and-results.html#results"><i class="fa fa-check"></i><b>2.4</b> Results:</a></li>
<li class="chapter" data-level="2.5" data-path="june2020-fin-clips-19-20-may-2020-lab-procedure-and-results.html"><a href="june2020-fin-clips-19-20-may-2020-lab-procedure-and-results.html#stored"><i class="fa fa-check"></i><b>2.5</b> Stored:</a></li>
<li class="chapter" data-level="2.6" data-path="june2020-fin-clips-19-20-may-2020-lab-procedure-and-results.html"><a href="june2020-fin-clips-19-20-may-2020-lab-procedure-and-results.html#follow-up"><i class="fa fa-check"></i><b>2.6</b> Follow up:</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="june2020-incubator-documentation-24well-plates.html"><a href="june2020-incubator-documentation-24well-plates.html"><i class="fa fa-check"></i><b>3</b> 11June2020 Incubator Documentation 24well plates</a><ul>
<li class="chapter" data-level="3.0.1" data-path="june2020-incubator-documentation-24well-plates.html"><a href="june2020-incubator-documentation-24well-plates.html#dish-a-from-2june2020"><i class="fa fa-check"></i><b>3.0.1</b> Dish A from 2June2020</a></li>
<li class="chapter" data-level="3.0.2" data-path="june2020-incubator-documentation-24well-plates.html"><a href="june2020-incubator-documentation-24well-plates.html#dish-b-from-2june2020"><i class="fa fa-check"></i><b>3.0.2</b> Dish B From 2June2020:</a></li>
<li class="chapter" data-level="3.0.3" data-path="june2020-incubator-documentation-24well-plates.html"><a href="june2020-incubator-documentation-24well-plates.html#big-dish-c-from-3june2020"><i class="fa fa-check"></i><b>3.0.3</b> Big dish C From 3June2020</a></li>
<li class="chapter" data-level="3.0.4" data-path="june2020-incubator-documentation-24well-plates.html"><a href="june2020-incubator-documentation-24well-plates.html#plate-b"><i class="fa fa-check"></i><b>3.0.4</b> 24plate B:</a></li>
<li class="chapter" data-level="3.0.5" data-path="june2020-incubator-documentation-24well-plates.html"><a href="june2020-incubator-documentation-24well-plates.html#no.55-from-8june2020"><i class="fa fa-check"></i><b>3.0.5</b> no.55 from 8june2020</a></li>
<li class="chapter" data-level="3.0.6" data-path="june2020-incubator-documentation-24well-plates.html"><a href="june2020-incubator-documentation-24well-plates.html#no.48-from-8june2020"><i class="fa fa-check"></i><b>3.0.6</b> no.48 from 8june2020</a></li>
<li class="chapter" data-level="3.0.7" data-path="june2020-incubator-documentation-24well-plates.html"><a href="june2020-incubator-documentation-24well-plates.html#inj3-09june2020"><i class="fa fa-check"></i><b>3.0.7</b> inj3 09june2020</a></li>
<li class="chapter" data-level="3.0.8" data-path="june2020-incubator-documentation-24well-plates.html"><a href="june2020-incubator-documentation-24well-plates.html#inj3-control-no-injection-no.79-9june2020"><i class="fa fa-check"></i><b>3.0.8</b> inj3 control (no injection) no.79 9june2020</a></li>
<li class="chapter" data-level="3.0.9" data-path="june2020-incubator-documentation-24well-plates.html"><a href="june2020-incubator-documentation-24well-plates.html#inj2-8june2020"><i class="fa fa-check"></i><b>3.0.9</b> inj2 8june2020</a></li>
<li class="chapter" data-level="3.1" data-path="june2020-incubator-documentation-24well-plates.html"><a href="june2020-incubator-documentation-24well-plates.html#inj2-control-8june2020"><i class="fa fa-check"></i><b>3.1</b> inj2 control 8june2020</a></li>
<li class="chapter" data-level="3.2" data-path="june2020-incubator-documentation-24well-plates.html"><a href="june2020-incubator-documentation-24well-plates.html#june2020"><i class="fa fa-check"></i><b>3.2</b> 16june2020</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="crisprcas9-mi-16june2020-ki.html"><a href="crisprcas9-mi-16june2020-ki.html"><i class="fa fa-check"></i><b>4</b> CRISPR/Cas9 MI 16june2020 KI</a><ul>
<li class="chapter" data-level="4.1" data-path="crisprcas9-mi-16june2020-ki.html"><a href="crisprcas9-mi-16june2020-ki.html#inj-ovservation-and-treatment"><i class="fa fa-check"></i><b>4.1</b> Inj ovservation and treatment:</a></li>
<li class="chapter" data-level="4.2" data-path="crisprcas9-mi-16june2020-ki.html"><a href="crisprcas9-mi-16june2020-ki.html#inj-ovservation-and-treatment-1"><i class="fa fa-check"></i><b>4.2</b> Inj ovservation and treatment:</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="june2020-fin-clips-19-20-may-2020-planning.html"><a href="june2020-fin-clips-19-20-may-2020-planning.html"><i class="fa fa-check"></i><b>5</b> 19June2020 Fin clips, 19-20 May 2020 - Planning</a><ul>
<li class="chapter" data-level="5.0.1" data-path="june2020-fin-clips-19-20-may-2020-planning.html"><a href="june2020-fin-clips-19-20-may-2020-planning.html#pcr-mixes"><i class="fa fa-check"></i><b>5.0.1</b> PCR mixes:</a></li>
<li class="chapter" data-level="5.0.2" data-path="june2020-fin-clips-19-20-may-2020-planning.html"><a href="june2020-fin-clips-19-20-may-2020-planning.html#etc-procedures"><i class="fa fa-check"></i><b>5.0.2</b> etc procedures:</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="learned-total-rna-extraction-from-hagai.html"><a href="learned-total-rna-extraction-from-hagai.html"><i class="fa fa-check"></i><b>6</b> Learned total RNA extraction from Hagai</a></li>
<li class="chapter" data-level="7" data-path="frankenbodies-arrival.html"><a href="frankenbodies-arrival.html"><i class="fa fa-check"></i><b>7</b> Frankenbodies arrival</a><ul>
<li class="chapter" data-level="7.1" data-path="frankenbodies-arrival.html"><a href="frankenbodies-arrival.html#frankenbodies-plasmids"><i class="fa fa-check"></i><b>7.1</b> Frankenbodies plasmids:</a></li>
<li class="chapter" data-level="7.2" data-path="frankenbodies-arrival.html"><a href="frankenbodies-arrival.html#frankenbodies-colony-seeding"><i class="fa fa-check"></i><b>7.2</b> Frankenbodies colony seeding</a></li>
</ul></li>
<li class="chapter" data-level="8" data-path="seeded-bacteria-containing-frankenbody-plasmids-fb.html"><a href="seeded-bacteria-containing-frankenbody-plasmids-fb.html"><i class="fa fa-check"></i><b>8</b> Seeded bacteria containing Frankenbody plasmids (FB)</a></li>
<li class="chapter" data-level="9" data-path="july2020-zfish-talk.html"><a href="july2020-zfish-talk.html"><i class="fa fa-check"></i><b>9</b> 15July2020 zFish Talk:</a></li>
<li class="chapter" data-level="10" data-path="july2020-zfish-talk-1.html"><a href="july2020-zfish-talk-1.html"><i class="fa fa-check"></i><b>10</b> 16July2020 zFish Talk:</a></li>
<li class="chapter" data-level="11" data-path="july2020-zfish-talk-2.html"><a href="july2020-zfish-talk-2.html"><i class="fa fa-check"></i><b>11</b> 17July2020 zFish Talk</a></li>
<li class="chapter" data-level="12" data-path="july2020-frankenbodies-plasmid-arrival-streaking-cultures.html"><a href="july2020-frankenbodies-plasmid-arrival-streaking-cultures.html"><i class="fa fa-check"></i><b>12</b> 27July2020 Frankenbodies plasmid arrival, streaking cultures</a></li>
<li class="chapter" data-level="13" data-path="july2020-procedure-colony-streaking-of-arrived-frankenbody-plasmids-colonies.html"><a href="july2020-procedure-colony-streaking-of-arrived-frankenbody-plasmids-colonies.html"><i class="fa fa-check"></i><b>13</b> 27July2020 Procedure: colony streaking of arrived frankenbody plasmids colonies</a><ul>
<li class="chapter" data-level="13.1" data-path="july2020-procedure-colony-streaking-of-arrived-frankenbody-plasmids-colonies.html"><a href="july2020-procedure-colony-streaking-of-arrived-frankenbody-plasmids-colonies.html#day-1-27july2020"><i class="fa fa-check"></i><b>13.1</b> Day 1 27July2020</a></li>
<li class="chapter" data-level="13.2" data-path="july2020-procedure-colony-streaking-of-arrived-frankenbody-plasmids-colonies.html"><a href="july2020-procedure-colony-streaking-of-arrived-frankenbody-plasmids-colonies.html#day-2-28july2020"><i class="fa fa-check"></i><b>13.2</b> Day 2 28July2020</a></li>
<li class="chapter" data-level="13.3" data-path="july2020-procedure-colony-streaking-of-arrived-frankenbody-plasmids-colonies.html"><a href="july2020-procedure-colony-streaking-of-arrived-frankenbody-plasmids-colonies.html#day-3-29july2020"><i class="fa fa-check"></i><b>13.3</b> Day 3 29July2020</a></li>
<li class="chapter" data-level="13.4" data-path="july2020-procedure-colony-streaking-of-arrived-frankenbody-plasmids-colonies.html"><a href="july2020-procedure-colony-streaking-of-arrived-frankenbody-plasmids-colonies.html#day-4-30july2020"><i class="fa fa-check"></i><b>13.4</b> Day 4 30July2020</a></li>
<li class="chapter" data-level="13.5" data-path="july2020-procedure-colony-streaking-of-arrived-frankenbody-plasmids-colonies.html"><a href="july2020-procedure-colony-streaking-of-arrived-frankenbody-plasmids-colonies.html#day-5-31july2020"><i class="fa fa-check"></i><b>13.5</b> Day 5 31July2020</a></li>
</ul></li>
<li class="chapter" data-level="14" data-path="aug2020-pcmv-15f11-ha-mch-figuring-out-how-to-make-it-go-ivt.html"><a href="aug2020-pcmv-15f11-ha-mch-figuring-out-how-to-make-it-go-ivt.html"><i class="fa fa-check"></i><b>14</b> 04Aug2020 - pCMV-15F11-HA-mCh: Figuring out how to make it go IVT</a></li>
<li class="chapter" data-level="15" data-path="thursday-to-09aug2020-pcdna3-1-dnm1-ha-plasmid-transfection-and-cloning.html"><a href="thursday-to-09aug2020-pcdna3-1-dnm1-ha-plasmid-transfection-and-cloning.html"><i class="fa fa-check"></i><b>15</b> Thursday to 09Aug2020 - pcDNA3.1-DNM1-HA plasmid transfection and cloning</a></li>
<li class="chapter" data-level="16" data-path="aug2020-dynamin-ha-plasmid-digestion.html"><a href="aug2020-dynamin-ha-plasmid-digestion.html"><i class="fa fa-check"></i><b>16</b> 10Aug2020 - Dynamin-HA plasmid digestion</a></li>
<li class="chapter" data-level="17" data-path="streaking-colony-isolation.html"><a href="streaking-colony-isolation.html"><i class="fa fa-check"></i><b>17</b> <span>Streaking</span> colony isolation:</a><ul>
<li class="chapter" data-level="17.1" data-path="streaking-colony-isolation.html"><a href="streaking-colony-isolation.html#materials"><i class="fa fa-check"></i><b>17.1</b> Materials:</a></li>
<li class="chapter" data-level="17.2" data-path="streaking-colony-isolation.html"><a href="streaking-colony-isolation.html#biological-hood-preparation"><i class="fa fa-check"></i><b>17.2</b> Biological hood preparation:</a></li>
<li class="chapter" data-level="17.3" data-path="streaking-colony-isolation.html"><a href="streaking-colony-isolation.html#antibiotics-preparations"><i class="fa fa-check"></i><b>17.3</b> Antibiotics preparations:</a></li>
<li class="chapter" data-level="17.4" data-path="streaking-colony-isolation.html"><a href="streaking-colony-isolation.html#plate-preparation"><i class="fa fa-check"></i><b>17.4</b> Plate preparation:</a></li>
<li class="chapter" data-level="17.5" data-path="streaking-colony-isolation.html"><a href="streaking-colony-isolation.html#streaking"><i class="fa fa-check"></i><b>17.5</b> Streaking:</a></li>
<li class="chapter" data-level="17.6" data-path="streaking-colony-isolation.html"><a href="streaking-colony-isolation.html#colony-isolation"><i class="fa fa-check"></i><b>17.6</b> Colony isolation</a></li>
<li class="chapter" data-level="17.7" data-path="streaking-colony-isolation.html"><a href="streaking-colony-isolation.html#glycerol-stock"><i class="fa fa-check"></i><b>17.7</b> Glycerol stock</a></li>
</ul></li>
<li class="chapter" data-level="18" data-path="streaking-colony-isolation-1.html"><a href="streaking-colony-isolation-1.html"><i class="fa fa-check"></i><b>18</b> <span>Streaking</span> colony isolation:</a><ul>
<li class="chapter" data-level="18.1" data-path="streaking-colony-isolation-1.html"><a href="streaking-colony-isolation-1.html#materials-1"><i class="fa fa-check"></i><b>18.1</b> Materials:</a></li>
<li class="chapter" data-level="18.2" data-path="streaking-colony-isolation-1.html"><a href="streaking-colony-isolation-1.html#biological-hood-preparation-1"><i class="fa fa-check"></i><b>18.2</b> Biological hood preparation:</a></li>
<li class="chapter" data-level="18.3" data-path="streaking-colony-isolation-1.html"><a href="streaking-colony-isolation-1.html#antibiotics-preparations-1"><i class="fa fa-check"></i><b>18.3</b> Antibiotics preparations:</a></li>
<li class="chapter" data-level="18.4" data-path="streaking-colony-isolation-1.html"><a href="streaking-colony-isolation-1.html#plate-preparation-1"><i class="fa fa-check"></i><b>18.4</b> Plate preparation:</a></li>
<li class="chapter" data-level="18.5" data-path="streaking-colony-isolation-1.html"><a href="streaking-colony-isolation-1.html#streaking-1"><i class="fa fa-check"></i><b>18.5</b> Streaking:</a></li>
<li class="chapter" data-level="18.6" data-path="streaking-colony-isolation-1.html"><a href="streaking-colony-isolation-1.html#colony-isolation-1"><i class="fa fa-check"></i><b>18.6</b> Colony isolation</a></li>
<li class="chapter" data-level="18.7" data-path="streaking-colony-isolation-1.html"><a href="streaking-colony-isolation-1.html#glycerol-stock-1"><i class="fa fa-check"></i><b>18.7</b> Glycerol stock</a></li>
</ul></li>
<li class="chapter" data-level="19" data-path="pfa-whole-mount-larvae-fixation.html"><a href="pfa-whole-mount-larvae-fixation.html"><i class="fa fa-check"></i><b>19</b> PFA whole -mount larvae fixation</a><ul>
<li class="chapter" data-level="19.1" data-path="pfa-whole-mount-larvae-fixation.html"><a href="pfa-whole-mount-larvae-fixation.html#materials-2"><i class="fa fa-check"></i><b>19.1</b> Materials:</a></li>
<li class="chapter" data-level="19.2" data-path="pfa-whole-mount-larvae-fixation.html"><a href="pfa-whole-mount-larvae-fixation.html#procedure"><i class="fa fa-check"></i><b>19.2</b> Procedure:</a></li>
</ul></li>
<li class="chapter" data-level="20" data-path="hit-dh5α-high-108-cat-no-rh618-by-real-biotech.html"><a href="hit-dh5α-high-108-cat-no-rh618-by-real-biotech.html"><i class="fa fa-check"></i><b>20</b> HIT-Dh5α High 10<sup>8</sup> Cat.No: RH618 by real-biotech</a><ul>
<li class="chapter" data-level="20.1" data-path="hit-dh5α-high-108-cat-no-rh618-by-real-biotech.html"><a href="hit-dh5α-high-108-cat-no-rh618-by-real-biotech.html#materials-3"><i class="fa fa-check"></i><b>20.1</b> Materials:</a></li>
<li class="chapter" data-level="20.2" data-path="hit-dh5α-high-108-cat-no-rh618-by-real-biotech.html"><a href="hit-dh5α-high-108-cat-no-rh618-by-real-biotech.html#procedure-1"><i class="fa fa-check"></i><b>20.2</b> Procedure:</a><ul>
<li class="chapter" data-level="20.2.1" data-path="hit-dh5α-high-108-cat-no-rh618-by-real-biotech.html"><a href="hit-dh5α-high-108-cat-no-rh618-by-real-biotech.html#antibiotics-preparations-2"><i class="fa fa-check"></i><b>20.2.1</b> Antibiotics preparations:</a></li>
<li class="chapter" data-level="20.2.2" data-path="hit-dh5α-high-108-cat-no-rh618-by-real-biotech.html"><a href="hit-dh5α-high-108-cat-no-rh618-by-real-biotech.html#plate-prep"><i class="fa fa-check"></i><b>20.2.2</b> Plate prep:</a></li>
<li class="chapter" data-level="20.2.3" data-path="hit-dh5α-high-108-cat-no-rh618-by-real-biotech.html"><a href="hit-dh5α-high-108-cat-no-rh618-by-real-biotech.html#transfer"><i class="fa fa-check"></i><b>20.2.3</b> Transfer:</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="21" data-path="introduction.html"><a href="introduction.html"><i class="fa fa-check"></i><b>21</b> Introduction</a></li>
<li class="chapter" data-level="22" data-path="t7-splicing-fbgfp-cmv-t7-insertion.html"><a href="t7-splicing-fbgfp-cmv-t7-insertion.html"><i class="fa fa-check"></i><b>22</b> T7 Splicing: FBgfp CMV T7 insertion</a><ul>
<li class="chapter" data-level="22.1" data-path="t7-splicing-fbgfp-cmv-t7-insertion.html"><a href="t7-splicing-fbgfp-cmv-t7-insertion.html#run-cut"><i class="fa fa-check"></i><b>22.1</b> run cut</a><ul>
<li class="chapter" data-level="22.1.1" data-path="t7-splicing-fbgfp-cmv-t7-insertion.html"><a href="t7-splicing-fbgfp-cmv-t7-insertion.html#on-strip"><i class="fa fa-check"></i><b>22.1.1</b> on strip:</a></li>
<li class="chapter" data-level="22.1.2" data-path="t7-splicing-fbgfp-cmv-t7-insertion.html"><a href="t7-splicing-fbgfp-cmv-t7-insertion.html#result"><i class="fa fa-check"></i><b>22.1.2</b> result</a></li>
</ul></li>
<li class="chapter" data-level="22.2" data-path="t7-splicing-fbgfp-cmv-t7-insertion.html"><a href="t7-splicing-fbgfp-cmv-t7-insertion.html#clean"><i class="fa fa-check"></i><b>22.2</b> clean</a><ul>
<li class="chapter" data-level="22.2.1" data-path="t7-splicing-fbgfp-cmv-t7-insertion.html"><a href="t7-splicing-fbgfp-cmv-t7-insertion.html#results-1"><i class="fa fa-check"></i><b>22.2.1</b> results:</a></li>
</ul></li>
<li class="chapter" data-level="22.3" data-path="t7-splicing-fbgfp-cmv-t7-insertion.html"><a href="t7-splicing-fbgfp-cmv-t7-insertion.html#run-clean"><i class="fa fa-check"></i><b>22.3</b> run clean</a></li>
<li class="chapter" data-level="22.4" data-path="t7-splicing-fbgfp-cmv-t7-insertion.html"><a href="t7-splicing-fbgfp-cmv-t7-insertion.html#anneal"><i class="fa fa-check"></i><b>22.4</b> anneal</a><ul>
<li class="chapter" data-level="22.4.1" data-path="t7-splicing-fbgfp-cmv-t7-insertion.html"><a href="t7-splicing-fbgfp-cmv-t7-insertion.html#annealing-buffer"><i class="fa fa-check"></i><b>22.4.1</b> Annealing buffer:</a></li>
</ul></li>
<li class="chapter" data-level="22.5" data-path="t7-splicing-fbgfp-cmv-t7-insertion.html"><a href="t7-splicing-fbgfp-cmv-t7-insertion.html#ligation"><i class="fa fa-check"></i><b>22.5</b> Ligation</a><ul>
<li class="chapter" data-level="22.5.1" data-path="t7-splicing-fbgfp-cmv-t7-insertion.html"><a href="t7-splicing-fbgfp-cmv-t7-insertion.html#ligation-rxns-strip"><i class="fa fa-check"></i><b>22.5.1</b> ligation rxns (strip)</a></li>
<li class="chapter" data-level="22.5.2" data-path="t7-splicing-fbgfp-cmv-t7-insertion.html"><a href="t7-splicing-fbgfp-cmv-t7-insertion.html#x1-ligation-rxn-13"><i class="fa fa-check"></i><b>22.5.2</b> X1 ligation rxn/ 1:3</a></li>
<li class="chapter" data-level="22.5.3" data-path="t7-splicing-fbgfp-cmv-t7-insertion.html"><a href="t7-splicing-fbgfp-cmv-t7-insertion.html#x7-ligation-rxn-change-to-4"><i class="fa fa-check"></i><b>22.5.3</b> x7 ligation rxn CHANGE TO 4</a></li>
</ul></li>
<li class="chapter" data-level="22.6" data-path="t7-splicing-fbgfp-cmv-t7-insertion.html"><a href="t7-splicing-fbgfp-cmv-t7-insertion.html#transformation"><i class="fa fa-check"></i><b>22.6</b> Transformation:</a></li>
</ul></li>
<li class="chapter" data-level="23" data-path="fbgfp-after-t7-splicing-gel-analysis.html"><a href="fbgfp-after-t7-splicing-gel-analysis.html"><i class="fa fa-check"></i><b>23</b> FBgfp After T7 splicing, gel analysis</a></li>
<li class="chapter" data-level="24" data-path="danger-danger-high-voltage-the-experiment-f-a-i-l-e-d.html"><a href="danger-danger-high-voltage-the-experiment-f-a-i-l-e-d.html"><i class="fa fa-check"></i><b>24</b> Danger danger - High Voltage - The experiment F A I L E D</a></li>
<li class="chapter" data-level="25" data-path="new-digestions.html"><a href="new-digestions.html"><i class="fa fa-check"></i><b>25</b> New Digestions:</a></li>
<li class="chapter" data-level="26" data-path="fbgfp-after-t7-splicing-gel-analysis-1.html"><a href="fbgfp-after-t7-splicing-gel-analysis-1.html"><i class="fa fa-check"></i><b>26</b> FBgfp After T7 splicing, gel analysis</a></li>
<li class="chapter" data-level="27" data-path="t4-and-re-qa.html"><a href="t4-and-re-qa.html"><i class="fa fa-check"></i><b>27</b> T4 and RE QA:</a><ul>
<li class="chapter" data-level="27.1" data-path="t4-and-re-qa.html"><a href="t4-and-re-qa.html#gel-0.8-120v-35-min"><i class="fa fa-check"></i><b>27.1</b> Gel 0.8%, 120V, 35 min:</a></li>
</ul></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">A Minimal Book Example</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="header">
<h1 class="title">A Minimal Book Example</h1>
<p class="author"><em>Yihui Xie</em></p>
<p class="date"><em>2020-10-31</em></p>
</div>
<div id="prerequisites" class="section level1">
<h1><span class="header-section-number">Chapter 1</span> Prerequisites</h1>
<p>This is a <em>sample</em> book written in <strong>Markdown</strong>. You can use anything that Pandoc’s Markdown supports, e.g., a math equation <span class="math inline">\(a^2 + b^2 = c^2\)</span>.</p>
<p>The <strong>bookdown</strong> package can be installed from CRAN or Github:</p>
<div class="sourceCode" id="cb1"><pre class="sourceCode r"><code class="sourceCode r"><a class="sourceLine" id="cb1-1" title="1"><span class="kw">install.packages</span>(<span class="st">&quot;bookdown&quot;</span>)</a>
<a class="sourceLine" id="cb1-2" title="2"><span class="co"># or the development version</span></a>
<a class="sourceLine" id="cb1-3" title="3"><span class="co"># devtools::install_github(&quot;rstudio/bookdown&quot;)</span></a></code></pre></div>
<p>Remember each Rmd file contains one and only one chapter, and a chapter is defined by the first-level heading <code>#</code>.</p>
<p>To compile this example to PDF, you need XeLaTeX. You are recommended to install TinyTeX (which includes XeLaTeX): <a href="https://yihui.org/tinytex/" class="uri">https://yihui.org/tinytex/</a>.</p>

<p>Taken from V1.0 - Protocols: CRISPR/Cas9 (with HD) zebrafish zygote microinjection protocol</p>
<div id="nov2019-crisprcas9-with-hd-zebrafish-zygote-microinjection-protocol" class="section level2">
<h2><span class="header-section-number">1.1</span> 13Nov2019 CRISPR/Cas9 (with HD) zebrafish zygote microinjection protocol:</h2>
<!-- \hypertarget{protocol1}{protocol number 1} -->
<p><strong>ssODN dilution:</strong></p>
<p>Stock ssODN (M743T HD) is concentrated to 5932 ng/µg, dilution is required for injection mix preperation.</p>
<p>Mix material by ’s instructions on . Afterwards carry out vortex and spindown.</p>
<p><strong>Table 1 - ssODN dilution:</strong></p>

<p><strong>Cas9 protein dilution:</strong>
Stock Cas9 (protein) is concentrated to 5000 ng/µg, dilution is required for injection mix preperation.</p>
<p>Mix material by ’s instructions on . Afterwards carry out vortex and spindown.</p>
<p><strong>Table 2:</strong></p>
<table style="width:100%;">
<colgroup>
<col width="11%" />
<col width="25%" />
<col width="17%" />
<col width="6%" />
<col width="23%" />
<col width="15%" />
</colgroup>
<thead>
<tr class="header">
<th>Material</th>
<th>Initial Concentration</th>
<th>Volume to pick</th>
<th>DDW</th>
<th>final concontration</th>
<th>Final Volume</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>Cas9</td>
<td>5000 ng/µL</td>
<td>1 µL</td>
<td>4 µL</td>
<td>1000 ng/µg</td>
<td>12 µL</td>
</tr>
</tbody>
</table>
<p><strong>Cas9/gRNA complex formation</strong></p>
<p>Mix:</p>
<ul>
<li>DDW from  microinjection mix.</li>
<li>Cas9 from  microinjection mix.</li>
<li>gRNA from  microinjection mix.</li>
</ul>
<p>Place in incubation at 37°C for ~10 miutes.</p>
<p><strong>Injection mix:</strong></p>
<p>On , mix the following materials. Perform vortex and spindown.</p>
<strong>Table 3:</strong>

<p><strong>Scr7 Dilution:</strong></p>
<p>In a 50 ml falcon tube:</p>
<p><strong>Table 4:</strong></p>

<p><strong>DMSO Dilution:</strong></p>
<p>In a 50 ml falcon tube:</p>
<p><strong>Table 5:</strong></p>

<p><strong>Injection:</strong></p>
<ul>
<li>After zf mate - TL type, injection volume ~2 nL of injection mix (table 3), inject embryos at 1-cell stage on agarose gel track plate (pre heated to 28.5°C).</li>
<li>Keep a few embryos uninjected for negative control.</li>
<li>After injection move to Scr7 medium.</li>
</ul>

</div>
<div id="microinjections-on-november-13th" class="section level2">
<h2><span class="header-section-number">1.2</span> Microinjections on November 13<sup>th</sup>:</h2>
<p><em>Date: November 13<sup>th</sup>, 2019</em>
Following protocol <!-- \hyperlink{protocol1}{{\color{purple}{\underline{number 1}}}}. --></p>
<p>We got our  which I now call .</p>
<div id="batch-table" class="section level3">
<h3><span class="header-section-number">1.2.1</span> Batch table:</h3>

<p>Batch 1 compiles of 40 zygotes, 27 of which were microinjected by Hagai, 13 of which were injected by me.
See <a href="images/batch1TL_M743T_MI_Tom_nov_13_2019.jpg"></a>, or <a href="images/batch1TL_M743T_MI_hagai_nov_13_2019.jpg"></a></p>
<p>Due to distance between our lab and lab 3 where the microinjections took place, along with unawareness of lack of devices such as 10 µL pipettes, we performed the injections on 8-16 cell stage zygotes (Hagai and I).</p>
<p>In my case some of the microinjections did not enter the cell, most were inside the egg close to the cell.</p>

</div>
</div>
<div id="microinjection-expertise-examination-tom-v-hagai" class="section level2">
<h2><span class="header-section-number">1.3</span> Microinjection expertise examination Tom V Hagai</h2>

<p><em>Date: December 10<sup>th</sup>, 2019</em>
After conducting Microinjections on November 13<sup>th</sup>, I have data about . Part of the experiment’s objectives was to examine my skills vs Hagai’s skills.</p>
<p><a href="tables/Tom_VS_Hagai_R1_Tom.csv"></a></p>
<p><a href="tables/Tom_VS_Hagai_R1_Hagai.csv"></a></p>
<div id="table-7" class="section level3">
<h3><span class="header-section-number">1.3.1</span> Table 7:</h3>
<table>
<thead>
<tr>
<th style="text-align:left;">
Date checked
</th>
<th style="text-align:left;">
Time of the day
</th>
<th style="text-align:right;">
Live count
</th>
<th style="text-align:right;">
Dead Count
</th>
<th style="text-align:right;">
Dead accumulated
</th>
<th style="text-align:right;">
Dead accumulated percentage
</th>
<th style="text-align:right;">
Time
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
13-Nov
</td>
<td style="text-align:left;">
Morning
</td>
<td style="text-align:right;">
13
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
0
</td>
</tr>
<tr>
<td style="text-align:left;">
13-Nov
</td>
<td style="text-align:left;">
Evening
</td>
<td style="text-align:right;">
13
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
0.00000
</td>
<td style="text-align:right;">
7
</td>
</tr>
<tr>
<td style="text-align:left;">
14-Nov
</td>
<td style="text-align:left;">
Morning
</td>
<td style="text-align:right;">
7
</td>
<td style="text-align:right;">
6
</td>
<td style="text-align:right;">
6
</td>
<td style="text-align:right;">
14.63415
</td>
<td style="text-align:right;">
17
</td>
</tr>
<tr>
<td style="text-align:left;">
14-Nov
</td>
<td style="text-align:left;">
Evening
</td>
<td style="text-align:right;">
6
</td>
<td style="text-align:right;">
7
</td>
<td style="text-align:right;">
13
</td>
<td style="text-align:right;">
31.70732
</td>
<td style="text-align:right;">
24
</td>
</tr>
<tr>
<td style="text-align:left;">
15-Nov
</td>
<td style="text-align:left;">
Morning
</td>
<td style="text-align:right;">
6
</td>
<td style="text-align:right;">
7
</td>
<td style="text-align:right;">
20
</td>
<td style="text-align:right;">
48.78049
</td>
<td style="text-align:right;">
41
</td>
</tr>
<tr>
<td style="text-align:left;">
15-Nov
</td>
<td style="text-align:left;">
Evening
</td>
<td style="text-align:right;">
6
</td>
<td style="text-align:right;">
7
</td>
<td style="text-align:right;">
27
</td>
<td style="text-align:right;">
65.85366
</td>
<td style="text-align:right;">
48
</td>
</tr>
<tr>
<td style="text-align:left;">
16-Nov
</td>
<td style="text-align:left;">
Morning
</td>
<td style="text-align:right;">
6
</td>
<td style="text-align:right;">
7
</td>
<td style="text-align:right;">
34
</td>
<td style="text-align:right;">
82.92683
</td>
<td style="text-align:right;">
55
</td>
</tr>
<tr>
<td style="text-align:left;">
16-Nov
</td>
<td style="text-align:left;">
Evening
</td>
<td style="text-align:right;">
6
</td>
<td style="text-align:right;">
7
</td>
<td style="text-align:right;">
41
</td>
<td style="text-align:right;">
100.00000
</td>
<td style="text-align:right;">
72
</td>
</tr>
</tbody>
</table>
</div>
<div id="table-8" class="section level3">
<h3><span class="header-section-number">1.3.2</span> Table 8:</h3>

</div>
<div id="graph-1" class="section level3">
<h3><span class="header-section-number">1.3.3</span> Graph 1:</h3>
<p><img src="LabNote_files/figure-html/unnamed-chunk-9-1.png" width="672" /></p>
</div>
<div id="graph-2" class="section level3">
<h3><span class="header-section-number">1.3.4</span> Graph 2:</h3>
<p><img src="LabNote_files/figure-html/unnamed-chunk-10-1.png" width="672" /></p>
</div>
<div id="statistical-analysis" class="section level3">
<h3><span class="header-section-number">1.3.5</span> Statistical analysis</h3>
<div id="df1" class="section level4">
<h4><span class="header-section-number">1.3.5.1</span> DF1:</h4>
<pre><code>##  Date checked Time of the day   Live count       Dead Count    Dead accumulated
##  13-Nov:2     Evening:4       Min.   : 6.000   Min.   :0.000   Min.   : 0.00   
##  14-Nov:2     Morning:4       1st Qu.: 6.000   1st Qu.:6.500   1st Qu.: 9.50   
##  15-Nov:2                     Median : 6.000   Median :7.000   Median :20.00   
##  16-Nov:2                     Mean   : 7.875   Mean   :5.857   Mean   :20.14   
##                               3rd Qu.: 8.500   3rd Qu.:7.000   3rd Qu.:30.50   
##                               Max.   :13.000   Max.   :7.000   Max.   :41.00   
##                                                NA&#39;s   :1       NA&#39;s   :1       
##  Dead accumulated percentage      Time      
##  Min.   :  0.00              Min.   : 0.00  
##  1st Qu.: 23.17              1st Qu.:14.50  
##  Median : 48.78              Median :32.50  
##  Mean   : 49.13              Mean   :33.00  
##  3rd Qu.: 74.39              3rd Qu.:49.75  
##  Max.   :100.00              Max.   :72.00  
##  NA&#39;s   :1</code></pre>
</div>
<div id="df2" class="section level4">
<h4><span class="header-section-number">1.3.5.2</span> DF2:</h4>
<pre><code>##  Date checked Time of the day   Live count      Dead Count    Dead accumulated
##  13-Nov:2     Evening:4       Min.   :12.00   Min.   : 1.00   Min.   : 1.00   
##  14-Nov:2     Morning:4       1st Qu.:12.00   1st Qu.:13.50   1st Qu.:20.50   
##  15-Nov:2                     Median :12.00   Median :15.00   Median :43.00   
##  16-Nov:2                     Mean   :16.00   Mean   :12.57   Mean   :43.43   
##                               3rd Qu.:17.75   3rd Qu.:15.00   3rd Qu.:65.50   
##                               Max.   :27.00   Max.   :15.00   Max.   :88.00   
##                                               NA&#39;s   :1       NA&#39;s   :1       
##  Dead accumulated percentage      Time      
##  Min.   :  1.136             Min.   : 0.00  
##  1st Qu.: 23.295             1st Qu.:14.50  
##  Median : 48.864             Median :32.50  
##  Mean   : 49.351             Mean   :33.00  
##  3rd Qu.: 74.432             3rd Qu.:49.75  
##  Max.   :100.000             Max.   :72.00  
##  NA&#39;s   :1</code></pre>

<p>Taken from V1.0 under High priority</p>
</div>
</div>
</div>
<div id="a1-dna-extraction-pcr-for-sequencing-tl-ab-exon-3-seq" class="section level2">
<h2><span class="header-section-number">1.4</span> A1 DNA extraction &amp; PCR for sequencing, TL AB exon 3 seq</h2>
<p><em>February 3<sup>rd</sup>, 2020</em></p>
<p><a href="documents/Scan_Feb_3_2020.pdf"></a></p>
</div>
<div id="exon-3-sequencing" class="section level2">
<h2><span class="header-section-number">1.5</span> Exon 3 sequencing</h2>
<p>Project students require exon 3 sequence of AB and TL ZF to modify their reations based on our strains’ SNPs.</p>
<ul>
<li>Single AB<sup>wt</sup> individual from Jan 30<sup>th</sup>, 2020 breeding; AB genotype - .</li>
<li>Single TL<sup>wt</sup>, jan 26<sup>th</sup>, 2020 breeding; D8 parents.</li>
</ul>
<p>Their positions on the strip [FCD#001 3.2] are:</p>
<table>
<thead>
<tr class="header">
<th align="right">1</th>
<th>2</th>
<th>3</th>
<th>4</th>
<th>5</th>
<th>6</th>
<th>7</th>
<th>8</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="right">x</td>
<td>AB</td>
<td>TL</td>
<td>x</td>
<td>x</td>
<td>x</td>
<td>x</td>
<td>x</td>
</tr>
</tbody>
</table>
<div id="larvae-dna-extraction-protocol" class="section level3">
<h3><span class="header-section-number">1.5.1</span> Larvae DNA extraction protocol:</h3>
<p>For larvae samples.</p>
<p>Place larvae in tube and let it fall asleep cooling on ice.</p>
<p>Drain water with pipettor.</p>
<p>Add 50 [µl] lysis buffer.</p>
<p>Incubate for 10 [min] at 95 [°C].</p>
<p>Add 2.5 [µl] Protein K.</p>
<p>Incubate for 2-16 [h] at 55 [°C].</p>
<p>Cool on ice.</p>
<ul>
<li>Placed the tubes in our lab’s PCR device. .</li>
</ul>
</div>
<div id="a1-fc-pcr---pcrfc001" class="section level3">
<h3><span class="header-section-number">1.5.2</span> A1 FC PCR - PCRFC#001</h3>
<p>Again, for project students, these are the samples they tested nested PCR on (inner R primer specific for my KI design).
I do believe these samples are from details established at 2017 and not my details.</p>
<ul>
<li>Took a 1:10 dilution from stock DNA extract made by Hagai on FC from A1 details (Are now in genotype, )</li>
<li>The following PCR established:</li>
</ul>
<p>PCR mix:</p>
<table>
<thead>
<tr class="header">
<th align="left">Content</th>
<th align="center">1X volume [µl]</th>
<th align="center">8X volume [µl]</th>
<th align="center">Concentration</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">DNA</td>
<td align="center">1</td>
<td align="center">8.2</td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="left">F1 primer (M743T outer)</td>
<td align="center">1</td>
<td align="center">8.2</td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="left">R1 primer (M743T outer)</td>
<td align="center">1</td>
<td align="center">8.2</td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="left">GoTaq</td>
<td align="center">12.5</td>
<td align="center">102.5</td>
<td align="center"></td>
</tr>
<tr class="odd">
<td align="left">DDW</td>
<td align="center">9.5</td>
<td align="center">77.9</td>
<td align="center"></td>
</tr>
<tr class="even">
<td align="left">Total</td>
<td align="center">25</td>
<td align="center">205</td>
<td align="center"></td>
</tr>
</tbody>
</table>
<p></p>
<p>PCR conditions: </p>
<p>In tubes:</p>
<table>
<thead>
<tr class="header">
<th align="center">1</th>
<th align="center">2</th>
<th align="center">3</th>
<th align="center">4</th>
<th align="center">5</th>
<th align="center">6</th>
<th align="center">7</th>
<th align="center">8</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="center">A1-1</td>
<td align="center">A1-2</td>
<td align="center">A1-3</td>
<td align="center">A1-4</td>
<td align="center">A1-5</td>
<td align="center">A1-6</td>
<td align="center">A1-7</td>
<td align="center">NC</td>
</tr>
</tbody>
</table>
</div>
<div id="preparations-for-sequencing" class="section level3">
<h3><span class="header-section-number">1.5.3</span> Preparations for sequencing:</h3>
<p>Preparing primers for sequencing:</p>
<table>
<thead>
<tr class="header">
<th align="left">Name</th>
<th align="center">DDW [µl]</th>
<th align="center">100 nM () Primer Volume[µl]</th>
<th align="left">Total</th>
<th align="center">Label for sequencers</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">ex12 345 R</td>
<td align="center">47.4</td>
<td align="center">2.5</td>
<td align="left">50</td>
<td align="center">R1</td>
</tr>
<tr class="even">
<td align="left">ex12 1F</td>
<td align="center">47.4</td>
<td align="center">2.5</td>
<td align="left">50</td>
<td align="center">F1</td>
</tr>
</tbody>
</table>
</div>
</div>
<div id="grna-ivt-w-gne-2020-project-team" class="section level2">
<h2><span class="header-section-number">1.6</span> gRNA IVT (/w GNE 2020 project team):</h2>
<p><em>Date: Jan 7<sup>th</sup>, 2020</em></p>
<p><strong>INSERT PROTOCOL</strong></p>
<p><strong>MISTAKES</strong></p>
<ul>
<li>No DNase was added before RNA production(word?) was made. Gel expresses the presence of template DNA.</li>
</ul>
<p><strong>DATA to collect:</strong></p>
<ul>
<li>Materials, cas and other catalog numbers.</li>
<li>Number of gRNA tubes.</li>
<li>Gel run-time:
<ul>
<li>70 min, 110V</li>
<li>2 8 well gels</li>
<li>0.495g TopVision</li>
<li>0.459g hy-labs</li>
<li>33 ml TBEX0.5</li>
<li>Gelred</li>
<li>5 µl loaded to gel</li>
</ul></li>
</ul>
<p><strong>Files:</strong>
* <a href="images/Master%202020-01-07_17h42m09s%20gRNA%20high%20conc.jpg"></a>
* <a href="images/Master%202020-01-07_17h42m09s%20gRNA%20low%20conc.jpg"></a>
* <a href="tables/jan07_2020_IVTgRNA_Nanodrop.xml"></a></p>

<!--
Taken from v1.0 notebook, under High priority

## March 5^th^ 2020 - PCR 

By Hagai's decree we are hereby complled to follow through rigorous polymerase chain reaction for 16 fin clip details of different zebra fishes that now lie within the incubator inside small containers numbered A5#1-A5#12. 

The ZF details are from A5 cell in the old large system, they were spawned at April 24^th^ 2019 to parents in cell E3 (old large system). The parents are "male AB tg mut" with "female AB +/-".

**ZF Detail FC sampling**

| Strip | Original system location  | Current ID | Strain | Gender | Notes                                          |
| ----- | ------------------------- | ---------- | ------ | ------ | -----                                          |
| A-1   | A5                        | A5 #1      | AB     | M      |                                                |
| A-2   | A5                        | A5 #2      | TL     | M      |                                                |
| A-3   | A5                        | A5 #3      | AB     | M      |                                                |
| A-4   | A5                        | A5 #4      | TL     | M      |                                                |
| A-5   | A5                        | A5 #5      | AB     | F      | Died                                           |
| A-6   | A5                        | A5 #6      | AB     | F      | Died                                           |
| A-7   | A5                        | A5 #7      | TL     | F      |                                                |
| A-8   | A5                        | A5 #8      | AB     | F      |                                                |
| B-1   | A5                        | A5 #9      | AB     | M      |                                                |
| B-2   | A5                        | A5 #10     | AB     | F      |                                                |
| B-3   | A5                        | A5 #11     | AB     | M      |                                                |
| B-4   | A5                        | A5 #12     | TL     | M      |                                                |
| B-5   |                           |            |        |        | Rescue-injected - take 2 µl from original conc |
| B-6   |                           |            |        |        | Rescue-injected - take 2 µl from original conc |
| B-7   |                           |            |        |        | Rescue-injected - take 2 µl from original conc |
| B-8   |                           |            |        |        | Rescue-control - take 2 µl from original conc  |

**Instructions:**

  1. Dilute strip A by $\frac{1}{10}$.
  2. Dilute new working Primers - 28F at 10µM, F1_2 at 10 µM and zfgne_Ex3_408R, each final vol. 100 µl. 10 µl primer, 90 µl DDW.
  2. Dilute strip B, 1-4 by $\frac{1}{10}$. - 1 µL DNA, 9 µL DDW.
  3. Generate PCR mix without DNA.
  4. PCR Strip A.1:
      1. A5#1
      2. A5#2
      3. A5#3
      4. A5#4
      5. A5#5
      6. A5#6
      7. A5#7
      8. A5#8
  5. PCR Strip B.1:
      1. A5#9
      2. A5#10
      3. A5#11
      4. A5#12
      5. Rescue-injected - take 2 µl from original conc
      6. Rescue-injected - take 2 µl from original conc
      7. Rescue-injected - take 2 µl from original conc
      8. Rescue-control - take 2 µl from original conc
  6. PCR Strip C - controls:
      1. NG
      2. Positive control /w DNA Ko, a cDNA from a knockout
      3. `-`
      4. `-`
      5. `-`
      6. `-`
      7. `-`
      8. `-`
  7. PCR: T~A~ 60.4°C, 35 cycles, F1_2wts [HL]
  8. Gel prep:
      1. mix 0.375 g agarose /w 25 ml TBE and 1.5 µL dye.
      2. Load 12 µl sample, 3µL M100bp.
      3. 110V, 40 min, 1.5%

    
    
  
  
    
**PCR Mix for 18 reaction**

\begin{table}[H]
\centering
\begin{tabular}{llrr}
\toprule
Material & Unit & X1 & X18\\
\midrule
\rowcolor{gray!6}  DNA & µl & 2.0 & 36\\
pF & µl & 1.0 & 18\\
\rowcolor{gray!6}  pR & µl & 1.0 & 18\\
F1.2 & µl & 1.0 & 18\\
\rowcolor{gray!6}  GoTaq & µl & 12.5 & 225\\
\addlinespace
DDW & µl & 7.5 & 135\\
\rowcolor{gray!6}  final volume & µl & 25.0 & 450\\
\bottomrule
\end{tabular}
\end{table}

**Gel loading table:**

\begin{table}[H]
\centering
\begin{tabular}{llllllllllllllllllll}
\toprule
X1 & X2 & X3 & X4 & X5 & X6 & X7 & X8 & X9 & X10 & X11 & X12 & X13 & X14 & X15 & X16 & X17 & X18 & X19 & X20\\
\midrule
\rowcolor{gray!6}  M & A1 & A2 & A3 & A4 & A5 & A6 & A7 & M & A8 & B1 & B2 & B3 & B4 & B5 & B6 & B7 & B8 & M & -\\
- & M & C1 & C2 & - & - & - & - & - & M & - & - & - & - & - & - & - & - & M & -\\
\bottomrule
\end{tabular}
\end{table}

**Results:**



![Gel Electroporesis Results For A5 details PCR for exon 13](images/A5Genotyping4Mar2020.png)
1.5% agarose gel, 110V for 40 min.

Second row ran outside of gel, no negative control or positive control. Samples are noted as A5#<sample number>, Negative control noted as NC, Positive control noted as PC. Samples A5#1, #2, #3, #6, #8, #10 and #11 show 2 bands (200 & 300 bp). Samples A5#4, #5, #7, #9 and #12 show a single band (300bp).

**Notes:**

Hagai’s primers were left at Nataly’s and Bar’s box - [see image](../images/hagaisprimerslocation4mar2020.jpg).

For future reference: can we throw these? (pile of DNA extract tubes from A5 large system spawn) - [see image](images/pileoftubes4mar2020.jpg)

KO DNA sample left on yellow plate inside fume hood (right bottom corner).

**March 5^th^ 2020 - Samples B-5 to B-8 repeat.**

*Hypothesis:* Either something blocks the PCR, be it excess DNA or lack of DNA. 

We try high dilution and high concentration of the prior DNA extract samples.

**Instructions:**

1. Prepare reaction mixes A and B as follows.
2. Dilute extracted DNA for injected details (Strip B, 6-8) to $\frac{1}{40}$ as described in Strip A table.
3. Prepare PCR as follows in Stip Z and F tables.

### Mix A:

\begin{table}[H]
\centering
\begin{tabular}{llrr}
\toprule
Material & Unit & X1 & X6\\
\midrule
\rowcolor{gray!6}  DNA & µl & 2.0 & 12\\
pF & µl & 1.0 & 6\\
\rowcolor{gray!6}  pR & µl & 1.0 & 6\\
F1.2 & µl & 1.0 & 6\\
\rowcolor{gray!6}  GoTaq & µl & 12.5 & 75\\
\addlinespace
DDW & µl & 7.5 & 45\\
\rowcolor{gray!6}  final volume & µl & 25.0 & 150\\
\bottomrule
\end{tabular}
\end{table}


### Mix B:

\begin{table}[H]
\centering
\begin{tabular}{llrr}
\toprule
Material & Unit & X1 & X4\\
\midrule
\rowcolor{gray!6}  DNA & µl & 5.0 & 20\\
pF & µl & 1.0 & 4\\
\rowcolor{gray!6}  pR & µl & 1.0 & 4\\
F1.2 & µl & 1.0 & 4\\
\rowcolor{gray!6}  GoTaq & µl & 12.5 & 50\\
\addlinespace
DDW & µl & 4.5 & 18\\
\rowcolor{gray!6}  final volume & µl & 25.0 & 100\\
\bottomrule
\end{tabular}
\end{table}
-->
<p><strong>Strip A:</strong></p>
<table>
<thead>
<tr class="header">
<th>A</th>
<th>8</th>
<th>7</th>
<th>6</th>
<th>5</th>
<th>4</th>
<th>3</th>
<th>2</th>
<th>1</th>
<th>Load Vol</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>Sample</td>
<td>Inj1 X1</td>
<td>Inj2 X1</td>
<td>Inj3 X1</td>
<td>InjC X1</td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr class="even">
<td>Sample</td>
<td>DDW</td>
<td>DDW</td>
<td>DDW</td>
<td>DDW</td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
</tbody>
</table>
<p><strong>Strip Z:</strong></p>
<table style="width:100%;">
<colgroup>
<col width="10%" />
<col width="10%" />
<col width="10%" />
<col width="10%" />
<col width="10%" />
<col width="10%" />
<col width="10%" />
<col width="10%" />
<col width="10%" />
<col width="10%" />
</colgroup>
<thead>
<tr class="header">
<th>Z</th>
<th>1</th>
<th>2</th>
<th>3</th>
<th>4</th>
<th>5</th>
<th>6</th>
<th>7</th>
<th>8</th>
<th>Load Vol</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>Sample</td>
<td><span class="math inline">\(\frac{1}{40}\)</span> Inj1</td>
<td><span class="math inline">\(\frac{1}{40}\)</span> Inj2</td>
<td><span class="math inline">\(\frac{1}{40}\)</span> Inj3</td>
<td><span class="math inline">\(\frac{1}{40}\)</span> InjC</td>
<td>NC</td>
<td>PC</td>
<td></td>
<td></td>
<td>2µL</td>
</tr>
<tr class="even">
<td>Mix</td>
<td>A</td>
<td>A</td>
<td>A</td>
<td>A</td>
<td></td>
<td></td>
<td></td>
<td></td>
<td>23µL</td>
</tr>
</tbody>
</table>
<p><strong>Strip F:</strong></p>
<table>
<thead>
<tr class="header">
<th>F</th>
<th>1</th>
<th>2</th>
<th>3</th>
<th>4</th>
<th>5</th>
<th>6</th>
<th>7</th>
<th>8</th>
<th>Load Vol</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>Sample</td>
<td>Inj1 X1</td>
<td>Inj2 X1</td>
<td>Inj3 X1</td>
<td>InjC X1</td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr class="even">
<td>Mix</td>
<td>B</td>
<td>B</td>
<td>B</td>
<td>B</td>
<td></td>
<td></td>
<td></td>
<td></td>
<td>20µL</td>
</tr>
</tbody>
</table>
<p><strong>Results:</strong></p>
<div class="figure">
<img src="images/mRNArescue5Mar2020.png" alt="Gel Electroporesis Results For mRNA injected embryos PCR for exon 13" />
<p class="caption">Gel Electroporesis Results For mRNA injected embryos PCR for exon 13</p>
</div>
<p>Inj# - injected embryow <number>, NC - Negarive control, PC - Positive control.</p>
<div id="gel-electroporesis-results-for-a5-details-pcr-for-exon-13" class="section level3">
<h3><span class="header-section-number">1.6.1</span> Gel Electroporesis Results For A5 details PCR for exon 13</h3>
<p>1.5% agarose gel, 110V for 40 min.</p>
<p>Second row ran outside of gel, no negative control or positive control. Samples are noted as <code>A5#&lt;sample number&gt;</code>, Negative control noted as <code>NC</code>, Positive control noted as <code>PC</code>.
Samples A5#1, #2, #3, #6, #8, #10 and #11 show 2 bands (200 &amp; 300 bp). Samples A5#4, #5, #7, #9 and #12 show a single band (300bp).</p>
<p><strong>NOTES:</strong></p>
<p>Hagai’s primers were left at Nataly’s and Bar’s box - <a href="../images/hagaisprimerslocation4mar2020.jpg"></a>.</p>
<p>For future reference: can we throw these? (pile of DNA extract tubes from A5 large system spawn) <a href="images/pileoftubes4mar2020.jpg"></a>.</p>
<p>KO DNA sample left on yellow plate inside fume hood (right bottom corner).</p>

<table>
<tbody>
<tr class="odd">
<td>10June2020</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
            </section>

          </div>
        </div>
      </div>

<a href="june2020-fin-clips-19-20-may-2020-lab-procedure-and-results.html" class="navigation navigation-next navigation-unique" aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["LabNote.pdf", "LabNote.epub"],
"toc": {
"collapse": "subsection"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
